sdk/core/src/services/anchoringHandler.ts

```diff
@@ -37,7 +44,7 @@ const computePreparedJoloParams = async ({ registry, keysService, nonce, additio
 
   const {
     body: { digestHex },
-  } = await registry.createAnchorTransaction({ nonce, did, didDocumentAddress })
+  } = await registry.createAnchorTransaction({ nonce, did, didDocumentAddress, origin })
```

Comment:
there are no such property in registry for  [createAnchorTransaction](https://gitlab.com/affinidi/foundational/affinity-registry/-/blob/main/src/did/did.dto.ts#L58)

Reply:
@anton-iskryzhytskyi Do we need to add this field to registry then?

Reply:
yes, did is created when we anchoring-did, not a transaction, it is old naming, and from my point of view it is not accurate.
https://affinity-registry.prod.affinity-project.org/api/redoc#tag/DID/operation/AnchorDid only when you need to send "origin"